### PR TITLE
cli: more specific error msgs on install

### PR DESCRIPTION
**Pull request by krantzinator**
Small PR to match the error messaging we give on `uninstall` if the `-platform` flag is missing.

### CODE REVIEW

**Review by mitchellh — APPROVED** [review]

File: internal/cli/install.go [hunk 1]
```diff
@@ -60,8 +60,17 @@ func (c *InstallCommand) Run(args []string) int {
...
+				"The -platform flag is required.",
+				terminal.WithErrorStyle(),
+			)
+
+			return 1
+		}
+
 		c.ui.Output(
-			"Error installing server into %s: invalid platform",
+			"Error installing server into %q: invalid platform",
```
Comment (briancain) [inline internal/cli/install.go:73]: This was a bit of feedback on my upgrade PR, so I think it makes sense to do this here too:
```suggestion
			"Error installing server into %q: unsupported platform",
```

**Review by briancain — CHANGES_REQUESTED** [review]

File: internal/cli/install.go (hunk 1 above)
Comment (krantzinator) [reply internal/cli/install.go:73]: I _swear_ i put the `%q` in there! Must have dropped it amidst some stashes 😄 Thanks!

Comment (krantzinator) [reply internal/cli/install.go]: Ignore my previous comment; the git suggestion diff confused me 😄

**Review by briancain — APPROVED** [review]
woohoo! :+1: thanks for the update :smile:

**Pull request merged**